c_binding/libata.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (C) 2016 Red Hat, Inc.
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
```

Comment:
@joehandzik A quick grep shows that @cathay4t has updated the copyright on the simc plugin which executes out of process and some stuff in the `c_binding` directory and the `python_binding/lsm/_clib.c` file which do not.  Those files should likely be LGPL 2.1 like the others.  Lets see what @cathay4t has to say about all this.  If we cannot get to a consensus among ourselves on what we want to do, I will consult our legal team for direction here.  In my opinion I think we should just change everything to LGPL 2.1 as it's simpler and more compatible with other licenses.